src/cowboy_session.erl

```diff
@@ -85,36 +85,41 @@ init([]) ->
 %% Internal functions
 %% ===================================================================
 
+get_cookie(#{ cookie := undefined } = Req) ->
+	case cowboy_req:parse_cookies(Req) of
+		[] ->
+			undefined;
+		List ->
+			{ok, Cookie_name} = ?CONFIG(cookie_name),
+			{_Key, Value} = lists:keyfind(Cookie_name, 1, List),
```

Comment:
If the cookie isn't set, this `lists:keyfind/3` would return `false`. Probably best not to crash here.